tests/test_grainstats.py

```diff
@@ -195,6 +196,6 @@ def test_grainstats_get_triangle_height(base_point_1, base_point_2, top_point, e
 
 
 @pytest.mark.parametrize("edge_points, expected", [([[0, 0], [0, 1], [1, 0], [1, 1]], (1.0, 1.4142135623730951))])
-def test_get_min_max_ferets(edge_points, expected) -> None:
+def test_get_mini_max_ferets(edge_points, expected) -> None:
```

Comment:
is there a reason for this name change? The actual function is `get_max_min_ferets` if that is better?

Reply:
Fat fingers and being in the wrong buffer. Thanks for highlighting.